<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectPopup</class>
 <widget class="QDialog" name="ConnectPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <widget class="QLineEdit" name="mIpLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mIpLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enemy's IP address:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mConnectBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>115</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="mStateLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>165</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter IP address...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
